satellite/gracefulexit/endpoint.go

```diff
@@ -348,7 +356,14 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 			}
 
 			var transferMsg *pb.SatelliteMessage
+			mon.IntVal("graceful_exit_final_pieces_failed").Observe(progress.PiecesFailed)
+			mon.IntVal("graceful_exit_final_pieces_succeess").Observe(progress.PiecesTransferred)
+
 			processed := progress.PiecesFailed + progress.PiecesTransferred
+			if processed > 0 {
+				mon.IntVal("graceful_exit_successful_bytes_transfer_ratio").Observe(progress.PiecesTransferred / processed)
```

Comment:
don't you mean something like `graceful_exit_successful_pieces_transfer_ratio`

Reply:
I changed the name.